src/lib/jar.js

```diff
@@ -73,6 +73,21 @@ var Jar = {
         var expires = '; expires=' + new Date(new Date().setYear(new Date().getFullYear() + 1)).toUTCString();
         var path = '; path=/';
         document.cookie = obj + expires + path;
+    },
+    getSignedValue: function (cookieName, signedValue, callback) {
```

Comment:
The naming here is counterintuitive: `getSignedValue` gets the *un*signed value associated with the key named `signedValue`?